src/bin/pg_autoctl/primary_standby.c

```diff
@@ -411,7 +411,17 @@ primary_add_standby_to_hba(LocalPostgresServer *postgres, char *standbyHostname,
 	PGSQL *pgsql = &(postgres->sqlClient);
 	PostgresSetup *postgresSetup = &(postgres->postgresSetup);
 	char hbaFilePath[MAXPGPATH];
-	char* authMethod = replicationPassword ? "md5" : "trust";
+	char* authMethod = NULL;
```

Comment:
I am more used the the following spelling of the same logic, what do you think?

``` c
    char *authMethod = "trust";

    if (replicationPassword)
    {
```

Also please write `char *` with a space before the star, rather than how you did it... if only to look like the rest of the whole code base.

Reply:
agreed, I had only changed the right part of the declaration. Now fixed * part.